### PR TITLE
support for datadog provider v3

### LINKED ISSUE #1: Update to DD 3.1 and TF 1.0
It would be great if you could do the update to both the Datadog and Terraform 1.0. 

ps. Issue templates would be a big addition for GitHub.

### PR DESCRIPTION
**Pull request by xp-1000**
resolve #1

### CODE REVIEW

**Comment by Aohzan** [comment]
Error raised by Terraform:
```
database/elasticsearch/monitors-elasticsearch.tf line 451 => An argument named "threshold_windows" is not expected here.
database/mysql/monitors-mysql.tf line 201 => An argument named "threshold_windows" is not expected here.
database/mysql/monitors-mysql.tf line 243 => An argument named "threshold_windows" is not expected here.
```

File: README.md [hunk 1]
```diff
@@ -14,27 +14,31 @@ This repository aims to provide a base of generic and pre configured monitors fo
...
+      version = ">= 3.1.0"
+    }
+  }
+  required_version = ">= 0.12.31"
 }
 
 ```
 
-Note: terraform 0.12.26+ is required as it the first version compatible with, but ignoring, the `source` argument for provider requirements introduced in terraform 0.13.0.
+_Note_: if you want to use Datadog provider v2 so you need to use version 3 of this repository.
```
Comment (pdecat) [inline README.md:34]:
```suggestion
_Note_: if you want to use Datadog provider v2, you need to use version 3 of this module.
```

Comment (xp-1000) [reply README.md:34]: it feels weird to speak about "module" whereas we are on the root README which is not a module no?

**Comment by xp-1000** [comment]
> You have to replace all `threshold_windows = {` by `monitor_threshold_windows {`

@Aohzan good catch, it should be good now

**Review by Aohzan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (pdecat) [reply README.md:34]: What about:
```suggestion
_Note_: if you want to use Datadog provider v2, you need to use version 3 of the modules in this repository.
```

File: README.md [hunk 2]
```diff
@@ -14,27 +14,31 @@ This repository aims to provide a base of generic and pre configured monitors fo
 
 ## Getting started
 
-### Terraform
+### Versions
 
-Here is the minimum version required to use these modules of integrations.
+Here is the minimum versions required to use these modules of integrations.
```
Comment (pdecat) [inline README.md:19]:
```suggestion
Here are the minimum versions required to use these modules of integrations.
```

File: caas/docker/versions.tf [hunk 3]
```diff
@@ -1,7 +1,8 @@
 terraform {
   required_providers {
     datadog = {
-      source = "terraform-providers/datadog"
+      source = "DataDog/datadog"
+      version = ">= 3.1.0"
     }
   }
   required_version = ">= 0.12.26"
```
Comment (pdecat) [inline caas/docker/versions.tf:8]: Didn't you mean to bump this for all modules?

```suggestion
  required_version = ">= 0.12.31"
```

File: README.md (hunk 1 above)
Comment (xp-1000) [reply README.md:34]: good for me !

File: caas/docker/versions.tf (hunk 3 above)
Comment (xp-1000) [reply caas/docker/versions.tf]: yes thanks, should be good

**Review by pdecat — APPROVED** [review]
LGTM!

**Pull request merged**